### DISCUSSION TITLE
sudo orangepi-config not found error

### DISCUSSION THREAD

**Discussion by aadil3e** [Joshua-Riek/ubuntu-rockchip#457] [Q&A]
Hi I just loaded up  the latest desktop ubuntu for orange pi 5 plus, but I cant access the sudo orangepi-config   please help

**Comment by Joshua-Riek** [comment]
That command does not exist, this is not the official ubuntu image provided by Orange Pi.

**Reply by aadil3e** [reply]
Is there an official Ubuntu version  ? I was following this guy : https://www.crosstalksolutions.com/orange-pi-5-simple-overview-and-installation-with-m-2-ssd/   
As I want to move my stuff from the Micrso SD to the NVME, as I cant seem to get the orange pi to pick up on my pc via the USB c connector to flash the NVME direct

**Reply by aadil3e** [reply]
oh I see my errors I was using the wrong link for the ubuntu images dam im a idiot
